frontend/src/app/components/enterprise/enterprise-modal/enterprise-trial-form/ee-trial-form.component.sass

```diff
@@ -1,2 +1,2 @@
-.ee-trial-email-error
+.op-ee-trial-email-error
```

Comment:
The file should have the name of this block

Reply:
For now, the file name should be the name of the block without op, as you said.